libs/chlorophyll/scss/index.scss

```diff
@@ -1,2 +1,5 @@
 @forward 'tokens';
 @use 'components';
+
+// used for v-angular dropdown
+@import '@angular/cdk/overlay-prebuilt.css';
```

Comment:
This is not ideal, since it forces everyone to include all of Chlorophyll to get this import. And many of the V-Angular components already bundles styles from Chlorophyll, so you end up with a lot of duplicated CSS if you do this (probably over 500kb of extra CSS)

Ideally, Angular would have a way to declare global style dependencies for a component, but AFAIK, there's no way to do that. So I think consumers will just have to add this import line manually. Maybe you can add a note about this in the dropdown stories?